docs/Overview.md

```diff
@@ -21,7 +21,8 @@ The Specification includes:
 
 IS-05 is intended to be used in conjunction with an [IS-04 NMOS Discovery & Registration](https://specs.amwa.tv/is-04) deployment; however it has been written in such a way to provide useful functionality even in the absence of such a system.
 
-The terms 'Node', 'Device', 'Sender' and 'Receiver' are used extensively in this documenation set. The [NMOS Technical Overview](https://specs.amwa.tv/nmos/main/docs/2.0._Technical_Overview.html) provides an outline of these terms, and IS-04 provides corresponding schema definitions.
+The terms 'Node', 'Device', 'Sender' and 'Receiver' are used extensively in this documentation set.
+The [NMOS Technical Overview](https://specs.amwa.tv/nmos/main/docs/2.0._Technical_Overview.html) and the [NMOS Glossary][Glossary] define these and other common terms that have specific meanings in NMOS. 
 
```

Comment:
```suggestion
The [NMOS Technical Overview](https://specs.amwa.tv/nmos/main/docs/Technical_Overview.html) and the [NMOS Glossary][Glossary] define these and other common terms that have specific meanings in NMOS. 

```

Reply:
Thanks, @peterbrightwell. That link presumably needs fixing in other specs and other branches, and will be broken in all previously tagged versions. 😒